firmware/.gitignore

```diff
@@ -0,0 +1,5 @@
+.mbedbuild
```

Comment:
nit: this file isn't needed. Base .gitignore should use wildcards . Format should just be `ignore-dir-name/`. I need to update this organization wide, so don't worry about it too much for now.